src/mame/skeleton/shoken_md06.cpp

```diff
@@ -138,20 +180,34 @@ void shoken_md06_state::petitlot(machine_config &config)
 	m_maincpu->set_addrmap(AS_IO, &shoken_md06_state::io_map);
 
 	RTC62423(config, "rtc", 0);
+}
+
+void shoken_md06_state::polarstar(machine_config &config)
+{
+	petitlot(config);
+	m_maincpu->set_addrmap(AS_IO, &shoken_md06_state::sound_io_map);
 
 	// sound hardware
 	SPEAKER(config, "mono").front_center();
 
-	// TODO: sound related ICs aren't present?
+	okim9810_device &oki(OKIM9810(config, "oki", 16_MHz_XTAL / 12));
+	oki.add_route(ALL_OUTPUTS, "mono", 1.0); //divider guessed
 }
 
-
 ROM_START( petitlot )
 	ROM_REGION( 0x10000, "maincpu", 0 )
-	ROM_LOAD( "md06_ver4_1.ic6", 0x00000, 0x10000, CRC(ebc81f10) SHA1(28ac52aeadfbf792da95c01b16fb88f7a5eb1d4e) ) // 1xxxxxxxxxxxxxxx = 0xFF
+	ROM_LOAD( "md06_ver4_1.ic6", 0x00000, 0x10000, CRC(ebc81f10) SHA1(28ac52aeadfbf792da95c01b16fb88f7a5eb1d4e) ) // 1xxxxxxxxxxxxxxx = 0xff
```

Comment:
this is romcmp's output in a comment, it isn't subject to MAME's preference for lowercase hex.